pelican/settings.py

```diff
@@ -660,3 +661,18 @@ def configure_settings(settings):
             continue            # setting not specified, nothing to do
 
     return settings
+
+
+def coerce_overrides(overrides):
+    coerced = {}
+    types_to_cast = {int, str}
+    for k, v in overrides.items():
+        if k not in overrides:
+            logger.warning('Override for unknown setting %s, ignoring' % k)
```

Comment:
Close but not exactly :). That's still formatting it beforehand. Note **`,`** instead of **`%`**
 
```python
logger.warning('Override for unknown setting %s, ignoring', k)
```

Reply:
d'oh